config.json

```diff
@@ -209,6 +209,7 @@
       "plugin": "FogOfWar",
       "enabled": false,
       "mode": 1,
+      "allowedForGameModes": [],
```

Comment:
Set all possible gamemodes as default value, for reference

Reply:
added